python/src/main/python/drivers/graphicsfuzz_piglit_converter.py

```diff
@@ -62,10 +64,11 @@
 UNIFORM_DEC = 'uniform'
 
 
```

Comment:
`get_json_properties` is annotated as returning a `List` but I think it returns a `Dict`

Reply:
Missed this when I was fixing it up for the original PR - whoops. Funnily enough, the JSON was already annotated correctly in the function that it was used for.